packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -91,73 +93,75 @@ function SidePanel({
 	const expandLabel = t('SIDEPANEL_EXPAND', { defaultValue: 'Expand' });
 	const collapseTitle = t('SIDEPANEL_COLLAPSE', { defaultValue: 'Collapse' });
 	const toggleButtonTitle = docked ? expandLabel : collapseTitle;
-
+	const Components = Inject.getAll(getComponent, { Action });
 	return (
 		<nav className={navCSS} role="navigation">
+			{injected('preul')}
```

Comment:
This is not explicit except if
* you read it right ^^ (pre-ul and not preul that doesn't mean anything :D)
* you know the implementation

I think we should name them something like top/beforeActions/actions/afterActions/bottom
We should document it. Add a readme on sidepanel with an example of all insertion points